org/allPRs.ts

```diff
@@ -107,6 +107,15 @@ export const rfc16 = async () => {
   }
 }
 
+// Warn PR authors if they assign more than one person to a PR
+// https://github.com/artsy/README/issues/177
+export const rfc177 = async () => {
```

Comment:
I don't think this needs to be async? We could update it, and its test file.

```suggestion
export const rfc177 = () => {
```

Reply:
:+1: good call! done